tests/nested/manual/grade-signed-cloud-init-testkeys/task.yaml

```diff
@@ -135,10 +132,6 @@ execute: |
   echo "And we can run things as the normal user"
   tests.nested exec_as normal-user ubuntu "sudo true"
 
-  # TODO: check that we got a serial assertion via the fakedevicesvc
```

Comment:
The other test was adding:
```
   tests.nested exec "sudo snap model --verbose --serial" | MATCH "serial:\s+7777"
```
should this be added here as well?

Reply:
I thought it was a bit overkill to test it in multiple places actually, but it's easy enough for me to add that to this test too

Reply:
added